gse_account/models/company.py

```diff
@@ -0,0 +1,11 @@
+# -*- coding: utf-8 -*-
+from odoo import fields, models, api
+
+
+class ResCompany(models.Model):
+    _inherit = "res.company"
+
+    use_custom_msg = fields.Boolean('Custom Invoice Messages', config_parameter='account.use_custom_msg')
```

Comment:
@iambenkis @MaganaAsiati 
Here I would call the string "Use Custom Message", make sense with a boolean..